src/qutip_qip/qiskit/utils/converter.py

```diff
@@ -152,21 +147,4 @@ def convert_qiskit_circuit_to_qutip(
         elif qiskit_instruction.name in _ignore_gates:
             pass
 
-        else:
-            if not allow_custom_gate:
-                raise RuntimeError(
-                    f"{qiskit_instruction.name} is not a \
-gate in qutip_qip. To simulate this gate using it's corresponding \
-unitary matrix, set allow_custom_gate=True."
-                )
-
-            unitary = np.array(Operator(qiskit_instruction))
-            qutip_circuit.user_gates[qiskit_instruction.name] = (
-                _make_user_gate(unitary, qiskit_instruction)
-            )
-            qutip_circuit.add_gate(
-                qiskit_instruction.name,
-                targets=_get_mapped_bits(qiskit_qregs, bit_map=qubit_map),
-            )
-
```

Comment:
I forgot how and why this part exited before. This is no longer needed? Or no longer supported?